packages/cloudflare/src/index.ts

```diff
@@ -69,7 +69,14 @@ export default function createIntegration(args?: Options): AstroIntegration {
 
 	const isModeDirectory = args?.mode === 'directory';
 	const functionPerRoute = args?.functionPerRoute ?? false;
-	const runtimeMode = args?.runtime ?? 'off';
+
+	let runtimeMode: { mode: 'off' | 'local' | 'remote'; persistTo: string } = {
+		mode: 'off',
+		persistTo: '',
+	};
+	if (typeof args?.runtime === 'object') runtimeMode = args?.runtime;
+	if (args?.runtime === 'local') runtimeMode = { mode: 'local', persistTo: './wrangler/state/v3' };
+	if (args?.runtime === 'remote') runtimeMode = { mode: 'remote', persistTo: '' };
 
```

Comment:
This is all needed for backwards compatibility, and it does not even catch all edge-cases. I would like to outsource something like this to a lib, any suggestion for one, or how to handle it cleaner?